openssl-sys/src/lib.rs

```diff
@@ -674,6 +674,9 @@ extern "C" {
     pub fn SSL_CTX_add_extra_chain_cert(ctx: *mut SSL_CTX, x509: *mut X509) -> c_long;
     #[link_name = "SSL_CTX_set_read_ahead_shim"]
     pub fn SSL_CTX_set_read_ahead(ctx: *mut SSL_CTX, m: c_long) -> c_long;
+    #[cfg(feature = "ecdh_auto")]
+    #[link_name = "SSL_CTX_set_ecdh_auto_shim"]
```

Comment:
At least on LibreSSL, `SSL_CTX_set_ecdh_auto_shim()` is a macro. Isn't a shim required in that case?